pkg/sfu/buffer/buffer.go

```diff
@@ -625,3 +653,9 @@ func (b *Buffer) GetAudioLevel() (float64, bool) {
 
 	return b.audioLevel.GetLevel()
 }
+
+// TODO : now we rely on stream tracker for layer change, dependency still
+// work for that too. Do we keep it unchange or use both methods?
+func (b *Buffer) OnMaxLayerChanged(fn func(int, int)) {
+	b.maxLayerChangedCB = fn
+}
```

Comment:
I think all of this has to be changed to be SSRC based (buffer, stream tracker). An SSRC can have multiple layers or single layer. So, the `Bitrates` (the 3x4 array should be moved to buffer, stream tracker) definition should be moved here.

In the special case of `Simulcast`, `sfu/receiver` should patch proper layer and distribute it.